hack/generate.sh

```
@@ -121,6 +121,7 @@ function generate_bundle() {
     cd ..
     kustomize build config/manifests | operator-sdk generate bundle -q --overwrite=true --output-dir ${BUNDLE_OUTPUT_DIR} ${BUNDLE_METADATA_OPTS}
     mv ${__root}/bundle.Dockerfile ${BUNDLE_OUTPUT_DIR}/bundle.Dockerfile && sed -i '/scorecard/d' ${BUNDLE_OUTPUT_DIR}/bundle.Dockerfile
+    sed -i 's/createdAt: .*/createdAt: ""/' ${BUNDLE_OUTPUT_DIR}/manifests/assisted-service-operator.clusterserviceversion.yaml
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)